src/confab/vendor/github.com/pivotal-cf-experimental/gomegamatchers

```diff
@@ -1 +1 @@
-Subproject commit e817d3bd9c25e2ecff62c4e64b3eae908e774aec
+Subproject commit 0ea0a869c846c2f2df9a2e3fdca6e4d31be99f0c
```

Comment:
The `BeAnOsIsNotExistError` PR has been merged in https://github.com/pivotal-cf-experimental/gomegamatchers, so the submodule should point to the latest master of that repo.